Tools/autotest/common.py

```diff
@@ -13007,8 +13007,10 @@ def GPSTypes(self):
             (6, "SBP", None, "SBP", 5, 'detected'),
             # (7, "SBP2", 9, "SBP2", 5),  # broken, "waiting for config data"
             (8, "NOVA", 15, "NOVA", 5, 'detected'),  # no attempt to auto-detect this in AP_GPS
+            (11, "GSOF", 11, "GSOF", 5, 'detected'),
             (19, "MSP", 19, "MSP", 32, 'specified'),  # no attempt to auto-detect this in AP_GPS
             # (9, "FILE"),
+
```

Comment:
Spurious whitespace

Reply:
Removed in 22248770d5d59b2e038fb4f28996d96ce92cc25d